docs/plugins.md

```diff
@@ -92,6 +92,7 @@ These plugins apply transformations to your code.
 - [pipeline-operator](plugin-proposal-pipeline-operator.md)
 - [private-methods](plugin-proposal-private-methods.md)
 - [throw-expressions](plugin-proposal-throw-expressions.md)
+- [partial-application](plugin-proposal-partial-application.md)
```

Comment:
Nit: can we keep this list alpha sorted?

Reply:
oops, fixed now 👍